src/open_inwoner/cms/cases/views/status.py

```diff
@@ -280,7 +373,12 @@ def handle_document_upload(self, request, form):
                         filename=file.name
                     ),
                 )
-                return HttpResponseRedirect(self.get_success_url())
+
+                return HttpResponseClientRedirect(
+                    reverse(
+                        "cases:case_detail", kwargs={"object_id": str(self.case.uuid)}
+                    )
+                )
```

Comment:
I think all the `HttpResponseClientRedirect`'s should be a `redirect()` shortcut.

Reply:
I was mistaken, this was a HTMX redirect that doesn't do reverse()